### PR TITLE
GH-39114: [JS] Fix Example Code

### LINKED ISSUE #39114: Reading arrow files client-side
### Describe the usage question you have. Please include as many useful details as  possible.

When running the example in the `js` folder [read_file.html](https://github.com/apache/arrow/blob/main/js/examples/read_file.html), it says: `Arrow.Table.from is not a function`.  Please advise - what is the correct way to read an arrow file in a browser?

### Component(s)

JavaScript

### PR DESCRIPTION
**Pull request by tdiprima**
### Rationale for this change

Code is broken.  The modification fixes issue #39114: Reading arrow files client-side.

### What changes are included in this PR?

Changed `Arrow.Table.from` to `Arrow.tableFromIPC`.

### Are these changes tested?

Yes.

### Are there any user-facing changes?

No.
* Closes: #39114

### CODE REVIEW

File: js/examples/read_file.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
 }
 reader.onload = function (evt) {
 
-  var arrowTable = Arrow.Table.from([new Uint8Array(evt.target.result)]);
+  var arrowTable = Arrow.tableFromIPC([new Uint8Array(evt.target.result)]);
```
Comment (domoritz) [inline js/examples/read_file.html:44]: Does this work? 

```suggestion
  var arrowTable = Arrow.tableFromIPC(evt.target!.result);
```

**Review by domoritz — COMMENTED** [review]
Thank you. Maybe we can make the code simpler. Could you test simpler versions?

File: js/examples/read_file.html (hunk 1 above)
Comment (tdiprima) [reply js/examples/read_file.html:44]: The suggested revision `Arrow.tableFromIPC(evt.target!.result)` does not work.  Console error: "Uncaught SyntaxError: missing ) after argument list".

Comment (tdiprima) [reply js/examples/read_file.html:44]: If we change it to:

```js
var arrowTable = Arrow.tableFromIPC(evt.target.result);
```

it will work.

Comment (domoritz) [reply js/examples/read_file.html:44]: Let's use that then.

Comment (tdiprima) [reply js/examples/read_file.html]: OK, done.

**Review by trxcllnt — APPROVED** [review]

**Pull request merged**